pymc4/distributions/distribution.py

```diff
@@ -2,8 +2,10 @@
 import copy
 from typing import Optional, Union, Any
 
+from absl import logging
```

Comment:
Do we need/want this as a dependency?

Reply:
Isn't the `tf.logging` replaced with `absl-py` in `TF 2.0`?